app/views/how-tos/filters.html

```diff
@@ -0,0 +1,108 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+Filters - NHS prototype kit
+{% endblock %}
+
+{% block beforeContent %}
+  {% include "how-tos/includes/breadcrumb.html" %}
+{% endblock %}
+
+{% block content %}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-three-quarters">
+    <h1>Filters</h1>
+    <p class="nhsuk-lede-text">
+      Change how answers are shown.
+      </p>
+
+    {{ insetText({ html: '
+    <p>
+      To use filters, you need to know how to
+      <a href="passing-data-page">pass data from page to page</a>.
+    </p>
+    ' })}}
```

Comment:
Think we should use inset text super sparingly.

```suggestion
    <p>
      To use filters, you need to know how to
      <a href="passing-data-page">pass data from page to page</a>.
    </p>
```

Reply:
yep, changed